pkgs/os-specific/linux/syslinux/default.nix

```diff
@@ -66,6 +66,7 @@ stdenv.mkDerivation {
     "DATADIR=$(out)/share"
     "MANDIR=$(out)/share/man"
     "PERL=perl"
+    "HEXDATE=0x00000000" # hardcode HEXDATE to improve reproducibility
```

Comment:
(From the previous change)

> It's was current time as of making this pull request, I could put something else, although worth noting the value is used for integrity checking, so 0x00000000 is non-ideal, but maybe I should use such a value.

I'm out of the loop, but how is it affecting integrity checking, and what makes 0x0 not ideal? Would another extremely arbitrary but non-zero value be better? e.g. 0x12345678, 0xABCDABCD or 0x11111111.

Reply:
Actually, I don't think that's an issue. Integrity checking tries to detect whether the memdisk image was written completely, and the value is intended to be random. This looks like it:

```
ldlinux_magic	dd LDLINUX_MAGIC
		dd LDLINUX_MAGIC^HEXDATE
```

However, I don't think this should cause practical issues as syslinux is specifically to be used for our ISO images. As such, I think 0x00000000 is fine.